users/views/_admin_menus.php

```diff
@@ -1,7 +1,7 @@
 <?php
 $db = DB::getInstance();
 $menus = $db->query("SELECT * FROM us_menus")->results();
-$count = $db->query("SELECT z_index as z, COUNT(*) z_index FROM us_menus GROUP BY z_index HAVING z_index > 1")->results();
+$count = $db->query("SELECT z_index, COUNT(*) as count FROM us_menus GROUP BY z_index HAVING z_index > 1")->results();
```

Comment:
I don't think this query does the same thing as the original one. Maybe `SELECT z_index as z, COUNT(*) as z_index FROM us_menus GROUP BY z_index HAVING z_index > 1`.

Reply:
I checked with chat gpt, and yours and mine are doing the same thing. Just the alias is different. I am using count, and yours is using z. I don't know if AI gave me the correct answer. Anyway, I will use your professor.